message/models/Message.model.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose")
+const { Schema, model } = mongoose
+
+const messageSchema = new Schema({
+    destination: String,
+    message: String,
+    number: Number,
+    status: {
+        type: String,
+        enum: ['PENDANT', 'SENT', 'CONFIRMED'],
+        default: 'PENDANT'
```

Comment:
What is the problem that you solving when you set it by default to pendant?Do you think null would make sense more?

Reply:
I let de default status as PENDANT because in case of any unexpected problem, the message would be stated as PENDANT, pointing to some error. Do you think it's better not to set it pendant a default?